src/rcx_tk/process_metadata_file.py

```diff
@@ -15,11 +15,12 @@ def read_file(file_path):
 
 def save_dataframe_as_tsv(df, file_path):
```

Comment:
Please include type annotations in your code.

```suggestion
def save_dataframe_as_tsv(df: pd.DataFrame, file_path: str) -> None:
```

Reply:
resolved by [65ef8c9](https://github.com/RECETOX/rcx-tk/pull/13/commits/65ef8c9f11b624803534989304c88f3a596de3f9)